src/optlang/util.py

```diff
@@ -134,6 +135,24 @@ def list_available_solvers():
         log.debug('CPLEX python bindings found at %s' % os.path.dirname(cplex.__file__))
     except Exception:
         log.debug('CPLEX python bindings not available.')
+
+    # OSQP can be provided by the base or CUDA package
+    solvers['OSQP'] = False
+    try:
+        import cuosqp
```

Comment:
`cuosqp` does not provide a linear system solver, so it does not cover all the functionality of `osqp` (see https://github.com/oxfordcontrol/osqp/issues/30 and https://github.com/oxfordcontrol/cuosqp/issues/2). I guess the user will know that, but it would be nice to document that (and also document `osqp` at [`installation.rst`](https://optlang.readthedocs.io/en/latest/installation.html#solvers)).

Reply:
I think that just means that the direct linear solver does not run on the GPU for now, but it still uses a linear solver.